robot/Cumulus/tests/browser/recurring_donations/zpause_consecutive_payments.robot

```diff
@@ -0,0 +1,74 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/RecurringDonationsPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+Suite Setup     Run keywords
+...             Enable RD2
+...             Open Test Browser
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+
+Setup Test Data
+        [Documentation]         Create a contact and recurring donation record of type open using API
+        ...                     Associate the recurring donation to the contact
+
+        ${NS} =             Get NPSP Namespace Prefix
+        Set Suite Variable  ${NS}
```

Comment:
Can you fix the indentation here ? This whole keyword is out of sync from the test case below

Reply:
done